app/controllers/api/account.php

```diff
@@ -2123,11 +2131,17 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
             'phrase' => !empty($phrase) ? $phrase : '',
             // TODO: remove unnecessary team variable from this email
             'team' => '',
+            'twitterUrl' => APP_SOCIAL_TWITTER,
+            'discordUrl' => APP_SOCIAL_DISCORD,
+            'githubUrl' => APP_SOCIAL_GITHUB_APPWRITE,
+            'termsUrl' => APP_EMAIL_TERMS_URL,
+            'privacyUrl' => APP_EMAIL_PRIVACY_URL,
         ];
 
         $queueForMails
             ->setSubject($subject)
             ->setPreview($preview)
+            ->setBodyTemplate(__DIR__ . '/../../config/locale/templates/email-auth.tpl')
```

Comment:
We cannot apply branded base everywhere; we should only do this for Console emails. For Console, when we need branded emails, let's use SMTP templates (overrides) like in this PR: https://github.com/appwrite/appwrite/pull/10501